orchagent/neighorch.cpp

```diff
@@ -402,17 +425,61 @@ bool NeighOrch::removeNextHop(const IpAddress &ipAddress, const string &alias)
 
     assert(hasNextHop(nexthop));
 
+    SWSS_LOG_INFO("Removing next hop %s", nexthop.to_string().c_str());
+
     gFgNhgOrch->invalidNextHopInNextHopGroup(nexthop);
 
     if (m_syncdNextHops[nexthop].ref_count > 0)
     {
-        SWSS_LOG_ERROR("Failed to remove still referenced next hop %s on %s",
-                       ipAddress.to_string().c_str(), alias.c_str());
+        SWSS_LOG_ERROR("Failed to remove still referenced next hop %s",
```

Comment:
Revert back to original message

Reply:
It is not appropriate to revert this log to previous, since it will not contain full NH description.

Reply:
@prsunny New NextHopKey::toLogString() function maintains the original output.

Reply:
@prsunny All logs are reverted to previous format.